src/components/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import { FontType } from 'styled-components'
+import { BaseButton } from './styles/Button.styled'
+
+type Props = {
+  text: string
+  isActive?: boolean
+  link?: string | undefined
+  htmlType?: 'button' | 'submit' | 'reset' | undefined
+  fontType: FontType
+}
+
+const defaultProps = {
+  isActive: false,
+  link: null,
+  htmlType: 'submit',
+}
+
+function Button(props: Props) {
+  const { isActive } = props
+  return (
+    <BaseButton
+      onClick={() => (props.link === null ? undefined : window.open(props.link, '_blank'))}
+      isActive={isActive}
+      type={props.htmlType}
+      fontType={props.fontType}
+    >
+      {props.text}
+    </BaseButton>
+  )
```

Comment:
Reference the [first PR](https://github.com/rhdevs/rhdevs-website/pull/10/commits/962f4a16d2a8e2615ec36e9e12effb5aef1f9556) for props.link never being null, 2nd for [eslint comment](https://github.com/rhdevs/rhdevs-website/pull/35/commits/ae0f20cc6a9796e3aa72e961cfd0a358b059c6ea)



```suggestion
return (
    <>
      {/* eslint-disable-next-line security/detect-non-literal-fs-filename */}
      <BaseButton
        onClick={() => window.open(props.link, '_blank')}
        isActive={isActive}
        type={props.htmlType}
        fontType={props.fontType}
      >
        {props.text}
      </BaseButton>
    </>
  )
```

Reply:
Resolved in commit 89a417454b92b2e91852393fd934bcaae18f181a.